pages/docs/index.html

```diff
@@ -3,5 +3,144 @@
 
 {{define "body"}}
 <article>
+  <h1>What is Silverblue</h1>
+
+  <p><em>Fedora Silverblue</em> is the desktop version of Fedora Atomic Host. It is still heavily
+  in development with only few external users. This project is actively maintained and is
+  ready for use by sophisticated and interested users, but not ready for widespread promotion.</p>
```

Comment:
I am not sure that sophisticated is the right word to use here, it sound a bit dismissive to me. I would propose "early adopter" (which would at least position it on the innovation bell curve we see everywhere for "innovation").

Reply:
This text is stolen from the Project Atomic website. We should update it to reflect that Silverblue is all about changing this...